modules/_header.sass

```diff
@@ -49,11 +49,15 @@ $toggle-width: 900px
         display: flex
         width: 100%
 
-        > li
-          padding: 15px
+        >li
```

Comment:
inconsistent usage of the `> tag` selector. The rest of the file includes a space after the `>` symbol